core/math/quick_hull.cpp

```diff
@@ -434,6 +443,18 @@ Error QuickHull::build(const Vector<Vector3> &p_points, Geometry::MeshData &r_me
 		}
 	}
 
+	if (_flag_warnings) {
+		if (warning_f) {
+			WARN_PRINT("QuickHull : !F");
+		}
+		if (warning_o_equal_e) {
+			WARN_PRINT("QuickHull : O == E");
+		}
+		if (warning_o) {
+			WARN_PRINT("QuickHull : O == nullptr");
+		}
+	}
```

Comment:
If doing this we might as well write understandable warnings :) Here they're no longer linked to the actual lines of code where `F`, `O`, and `E` are defined.

Reply:
I'm not actually sure exactly what these do represent without dissecting the old Quickhull in detail (this might need the original author). I'm hesitant to put in more verbose warnings if they are incorrect. This is just an attempt to reduce the warnings spam to one of each per hull.. they seem benign.